src/mlpack/methods/random_forest/extra_trees_main.cpp

```diff
@@ -0,0 +1,278 @@
+/**
+ * @file methods/random_forest/extra_trees_main.cpp
+ * @author Pranshu Srivastava
+ *
+ * A program to build and evaluate extra trees.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/core.hpp>
+#include <mlpack/methods/random_forest/random_forest.hpp>
+#include <mlpack/methods/random_forest/random_binary_numeric_split.hpp>
+#include <mlpack/methods/decision_tree/random_dimension_select.hpp>
+#include <mlpack/core/util/mlpack_main.hpp>
+using namespace mlpack;
+using namespace mlpack::tree;
+using namespace mlpack::util;
+using namespace mlpack::tree::ExtraTrees;
+using namespace std;
+using namespace arma;
+
+// Program Name.
+BINDING_NAME("Extra Trees Forest");
+
+// Short description.
+BINDING_SHORT_DESC(
+    "An implementation of the standard Extra Trees algorithm"
```

Comment:
```suggestion
    "An implementation of the Extra Trees algorithm"
```
I don't think Extra Trees is necessarily "standard" in the same way that random forests are. :+1: